### PR TITLE
chore: cleanup dead code

### PR DESCRIPTION
**Pull request by tgriesser**
- Closes N/A

### User facing changelog

N/A

### Additional details
Removing some dead code that I came across while working on other tickets

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/data-context/src/sources/HtmlDataSource.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class HtmlDataSource {
         )
       } catch (e) {
         err = e
-        await this.ctx.util.delayMs(1000)
+        await new Promise((resolve) => setTimeout(resolve, 1000))
```
Comment (flotwig) [inline packages/data-context/src/sources/HtmlDataSource.ts:38]: might be worth just using `Bluebird.timeout` here and elsewhere in this PR... so when we strip `Bluebird` out these timeouts are more easily discovered and replaced with a shared helper. Blue and I recently discussed this here: https://github.com/cypress-io/cypress/pull/21328#discussion_r865463268

**Review by flotwig — APPROVED** [review]
lovely lovely lovely

File: packages/data-context/src/sources/HtmlDataSource.ts (hunk 1 above)
Comment (tgriesser) [reply packages/data-context/src/sources/HtmlDataSource.ts:38]: I don't think we use Bluebird at all in the `data-context` package currently

Comment (flotwig) [reply packages/data-context/src/sources/HtmlDataSource.ts:38]: :man_shrugging:

**Pull request merged**